etc/containerpilot.json

```diff
@@ -21,8 +21,8 @@
   "coprocesses": [{{ if .CONSUL_AGENT }}
     {
       "command": ["/usr/local/bin/consul", "agent",
-                  "-data-dir=/data",
-                  "-config-dir=/config",
+                  "-data-dir=/var/lib/consul",
+                  "-config-dir=/etc/consul",
```

Comment:
It probably doesn't matter, but the original paths were to match the ones in the progrium/consul image we've been using.